### DISCUSSION TITLE
Play the audio only while the rfc card is on the reader

### DISCUSSION THREAD

**Discussion by capsload2** [MiczFlor/RPi-Jukebox-RFID#1357] [Q&A]
is it possible to configure the phoniebox that its only playing sounds or music while the card sits on the cardreader?

**Comment by s-martin** [comment]
Depending on the used reader please check

- For RC522 https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/1097#issuecomment-713535784
- For USB reader https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/Stop-on-removal-with-USB-RFID-Reader
